node-require/calculate.js

```diff
@@ -0,0 +1,14 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const divide = require('./divide');
+const multiply = require('./multiply');
+
+if (process.argv[3] === 'plus') {
+  console.log('result:', add(process.argv[2], process.argv[4]));
+} else if (process.argv[3] === 'minus') {
+  console.log('result:', subtract(process.argv[2], process.argv[4]));
+} else if (process.argv[3] === 'times') {
+  console.log('result:', multiply(process.argv[2], process.argv[4]));
+} else if (process.argv[3] === 'over') {
+  console.log('result:', divide(process.argv[2], process.argv[4]));
+}
```

Comment:
Since you are accessing `process.argv` in several places it would be better and easier to save those values into variables first. For example:
```JAVASCRIPT
const num1 = process.argv[2];
const num2 = process.argv[4];
const op = process.argv[3];
```